.travis.yml

```diff
@@ -11,7 +11,9 @@ install:
 # develop seems to be required by travis since 02/2013
   - if [ $TRAVIS_PYTHON_VERSION == "2.7" -o $TRAVIS_PYTHON_VERSION == "3.4" ]; then pip install pyparsing==2.4.7; fi
   - python setup.py build develop
-  - pip install nose coverage
+  - pip install nose coverage rosdep
+  - sudo `which rosdep` init
```

Comment:
From https://github.com/ros-infrastructure/rospkg/pull/133#discussion_r270950763
> Why is `which` needed here?

Honestly I took this portion from [ros-infrastructure/bloom/.travis.yml#L17](https://github.com/ros-infrastructure/bloom/blob/4617d742ccdd6eec4236e47f1148e5ad15a70a2a/.travis.yml#L17), which seems to have been added in [ros-infrastructure/bloom#221](https://github.com/ros-infrastructure/bloom/pull/221). I can only guess that it is needed as `sudo` doesn't know where the executable of `rosdep` is, which is installed into user's space by `pip`. Not yet super sure why `sudo` was/is needed on Travis CI though.